templates/4_crystallicum_dark_templates/postbit/postbit_classic.html

```diff
@@ -1,45 +1,94 @@
-{$ignore_bit}
-<a name="pid{$post['pid']}" id="pid{$post['pid']}"></a>
-<div class="post classic {$unapproved_shade}" style="{$post_visibility}" id="post_{$post['pid']}">
-<div class="post_flexible">
-<div class="post_author scaleimages">
-	<center><strong><span class="largetext">{$post['profilelink']}</span></strong> {$post['onlinestatus']}</center>
-	{$post['useravatar']}<br>
-	<div class="author_information">
-			
-			<span class="smalltext">
-				{$post['usertitle']}<br />
-				{$post['userstars']}
-				{$post['groupimage']}
-			</span>
-	</div>
-	<div class="author_statistics">
-		{$post['user_details']}
-	</div>
-</div>
-<div class="post_content">
+<script>
+	document.addEventListener('DOMContentLoaded', function() {
+	  // Get the author avatar element
```

Comment:
I know there's something like CSS variables that should allow you get this effect without JS.
...well, at least I THINK so.
Here's solution by chatGPT, so you can test it, but I think this (...or something like this) should be possibe:
```CSS
.author_avatar {
  --avatar-height: calc(100% + 10px); /* Adding 10px as per your original logic */
  height: auto; /* Ensures the avatar adjusts its height naturally */
}

.post_author {
  min-height: var(--avatar-height);
}
```

Reply:
It seems to be working without it - you can remove the script, we'll worry about it when issue happens again :D